### PR TITLE
Mm 46609

### PR DESCRIPTION
**Pull request by ctlaltdieliet**
#### Summary
Removing some Markdown in[ feature_discovery](https://github.com/mattermost/mattermost-webapp/blob/master/components/admin_console/feature_discovery/feature_discovery.tsx#L170)  
#### Ticket Link
https://mattermost.atlassian.net/browse/MM-46609
#### Release Note
```release-note
None
```

### CODE REVIEW

**Comment by ctlaltdieliet** [comment]
I didn't know if I should use the full url or the constants like [here](https://github.com/mattermost/mattermost-webapp/blob/dfc7c4b2c096d7e422c70e4c111f3cb09e166876/components/admin_console/feature_discovery/feature_discovery.tsx#L178)? 
@hmhealey

**Review by hmhealey — APPROVED** [review]

**Comment by hmhealey** [comment]
Looks like those URLs are the same, so yeah, we should probably use those constants here since it's the same file

File: components/admin_console/feature_discovery/feature_discovery.tsx [hunk 1]
```diff
@@ -194,9 +194,32 @@ export default class FeatureDiscovery extends React.PureComponent<Props, State>
                             }}
                         />
                     ) : (
-                        <FormattedMarkdownMessage
+                        <FormattedMessage
                             id='admin.feature_discovery.trial-request.accept-terms'
-                            defaultMessage='By clicking **Start trial**, I agree to the [Mattermost Software Evaluation Agreement](!https://mattermost.com/software-evaluation-agreement/), [Privacy Policy](!https://mattermost.com/privacy-policy/), and receiving product emails.'
+                            defaultMessage='By clicking <highlight>Start trial</highlight>, I agree to the <linkEvaluation>Mattermost Software Evaluation Agreement</linkEvaluation>, <linkPrivacy>Privacy Policy</linkPrivacy>.'
```
Comment (julmondragon) [inline components/admin_console/feature_discovery/feature_discovery.tsx:199]: Was `, and receiving product emails` intended to be removed from the message?

Comment (ctlaltdieliet) [reply components/admin_console/feature_discovery/feature_discovery.tsx]: Oops... copy paste error. Fixed it!

**Review by julmondragon — APPROVED** [review]
Looks good! Thanks @ctlaltdieliet 

Screen Shot 2022-08-26 at 2 19 53 PM

**Pull request merged**